kraken-build/pyproject.toml

```
@@ -1,75 +1,59 @@
 [build-system]
-build-backend = "poetry.core.masonry.api"
-requires = ["poetry-core"]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-# Package metadata
-# ----------------
+[tool.hatch.build.targets.wheel]
+packages = ["src/kraken/"]
 
-[tool.poetry]
-authors = ["Niklas Rosenstein <rosensteinniklas@gmail.com>"]
-classifiers = []
-description = ""
-keywords = []
-license = "MIT"
+[project]
 name = "kraken-build"
-packages = [
-  {include = "kraken/build", from = "src"},
-  {include = "kraken/common", from = "src"},
-  {include = "kraken/core", from = "src"},
-  {include = "kraken/std", from = "src"},
-]
-readme = "README.md"
 version = "0.44.1"
+description = "Kraken is an extensible build orchestration system."
+readme = "README.md"
+requires-python = ">=3.10,<3.14"
+license = "MIT"
+authors = [
+  {name = "Niklas Rosenstein", email = "rosensteinniklas@gmail.com"},
+]
+dependencies = [
+  "databind>=4.5.0,<5.0.0",
```

tip (please ignore): 
```suggestion
  "databind~=4.5",
  "dill~=0.3.8",
```